source/crud/write-operations/bulk.txt

```diff
@@ -358,6 +359,34 @@ see the following API documentation:
 - `BulkWriteOptions <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/BulkWriteOptions.html>`__
 - `ordered() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/BulkWriteOptions.html#ordered(boolean)>`__
 
+
+Bulk Write Example: Full File
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/crud/example-intro.rst
+
+The following code is a complete, standalone file that performs the following
+actions:
+
+#. Creates a list of instances of the ``InsertOneModel``, ``UpdateOneModel``,
+   ``DeleteOneModel``, and ``ReplaceOneModel`` classes.
+#. Runs an ordered ``bulkWrite()`` operation on the list.
```

Comment:
S: I think it's more accurate to say something like "runs the writes specified in the list of models", rather than runs "on the list"
```suggestion
#. Runs an ordered ``bulkWrite()`` operation that performs the writes specified in the model list.
```